Gemfile

```diff
@@ -63,3 +63,5 @@ group :production do
   gem 'bugsnag'
   gem 'rack-google-analytics'
 end
+
+gem "redcarpet", "~> 3.5"
```

Comment:
can we switch to using the `commonmarker` gem as it has better support for github flavoured markdown, here's an example when i added it to one of my other projects: https://github.com/octobox/octobox/commit/3f44c654275b5d82217f7899ab07cd3d193b0984

Reply:
Thank you for the example link! I've updated it to use that gem instead.